binaries/checksums.txt

```diff
@@ -1,19 +0,0 @@
-# md5
-82131a43a72216c4ee109e06b4a0b52d arduino-firmware.bin
-ddb698057f864863bb77f57c976e4ee0 arduino-firmware.elf
-332fde9e5c4f4cdf268fb2a37016f244 circuitpython-scripts.bin
-1fa3a8d546c7b961256e3bece46b8caf circuitpython-swan_r5-nobootloader.bin
-ca5b529b7a7f79e5764a92c8de7869ad circuitpython.binpack
```

Comment:
I couldn't tell if these checksums were in the README, or if they had been moved elsewhere in this commit.

They seem important, because these binaries are our "official" offering for the time being.

Reply:
They were in a checksums.txt file. I'm not sure where most of these binaries came from.  I believe they were there so Brandon could give a quick demo without all the compilation. 

I've tried to keep the binaries minimal - anything that can be built from the repo is excluded, preferring build instructions instead. 

If there are reasons for keeping the binaries then I can reinstate them, with a readme about how they were built.

Can I naively ask why need checksums for binaries delivered over https?

cc: @bsatrom